src/javascript/app/pages/trade/process.js

```diff
@@ -63,9 +64,9 @@ const Process = (() => {
                 commonTrading.displayMarkets();
                 processMarket();
             } else if (country === 'gb' || country === 'im'){
-                $('#content').empty().html($('<div/>', { class: 'container' }).append($('<p/>', { class: 'notice-msg center-text', text: localize('Sorry, options trading isn’t available in the United Kingdom and the Isle of Man.') })));
+                NotAvailable.init({ body: 'Sorry, options trading isn’t available in the United Kingdom and the Isle of Man.' });
```

Comment:
please localize

Reply:
I localize it in the component. The props will pass into the component and localize